lib/bing-ads-reporting/downloader.rb

```diff
@@ -0,0 +1,27 @@
+module BingAdsReporting
+  class Downloader
+    include BingHelper
+
+    attr_reader :report_url
+
+    def initialize(report_url)
+      @report_url = report_url
+    end
+
+    def self.fetch_report(report_url)
+      new(report_url).fetch_report
+    end
+
+    def fetch_report
+      # logger.debug "Downloading Bing report from: #{report_url}"
```

Comment:
any reason for this being here commented out?

Reply:
good catch. I'll remove the comment.

Reply:
done.